db/data_migration/20170818102913_send_missing_people_to_rummager.rb

```diff
@@ -0,0 +1,32 @@
+slugs = %w(
+  nick-hurd
+  nick-gibb
+  damian-green
+  brandon-lewis
+  elizabeth-truss
+  greg-hands
+  claire-perry
+  andrea-leadsom
+  lord-ashton-of-hyde
+  rory-stewart
+  stephen-barclay
+  richard-harrington
+  caroline-nokes
+  lord-oshaughnessy
+  david-rutley
+  rebecca-harris
+  mike-freer
+  nigel-adams
+  stuart-andrew
+  craig-whittaker
+  ian-duncan
+  david-hall--2
+  david-reid
+  duncan-thompson--2
+  david-reed
+  ben-merrick
+)
+
+missing_people = Person.where(slug: slugs)
+
```

Comment:
I'd add a return statement here, to allow the migration to run even if there is no data.

e.g. `return unless missing_people`

Apart from that, 👍